examples/model_selection/plot_learning_curve.py

```diff
@@ -70,20 +71,20 @@ def plot_learning_curve(estimator, title, X, y, axes=None, ylim=None, cv=None,
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validators that can be used here.
 
-    n_jobs : int or None, optional (default=None)
+    n_jobs : int or None, default=None
         Number of jobs to run in parallel.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    train_sizes : array-like, shape (n_ticks,), dtype float or int
+    train_sizes : array-like of shape (n_ticks,), dtype=float, int
```

Comment:
Do we have a documentation guideline for several `dtype` @glemaitre?

I do not know if I prefer `dtype=float or int` instead 😕.

Reply:
Actually that wasn't intended, I am not insisting on this change. As far as I see, there is no rule in the documentation guidelines on how to specify multiple data types.

Reply:
`dtype={int, float}` I think form previous review

Reply:
Ok, should I add it as an example in the documentation guidelines?

Reply:
yes we could do that in a separate PR.